google-style.yml

```diff
@@ -31,6 +31,8 @@ Metrics/LineLength:
   Max: 120
 Metrics/MethodLength:
   Max: 25
+Metrics/ModuleLength:
```

Comment:
Disabling this cop seems like an exception rather than a rule. Individual gems are going always going to be able to customize or make exceptions, but I don't know if the default should be to disable this.

Reply:
True. What are your thoughts on `Metrics/ClassLength:` being disabled?

Reply:
I didn't realize `Metrics/ClassLength` was disabled. Like `Metrics/ModuleLength`, that seems like a project-specific setting, not a general rule.

Reply:
I re-enabled both